tests/js/server/aql/aql-graph-k-shortest-path.js

```diff
@@ -524,8 +524,24 @@ function kAttributeWeightShortestPathTestSuite() {
 
 }
 
+function kShortestPathsSyntaxTestSuite() {
+  return {
+    testTooManyVariables: function () {
+      const query = `
+        FOR a, b IN OUTBOUND K_SHORTEST_PATHS "x" TO "y" GRAPH "G" RETURN a`;
+      try {
+        db._query(query).toArray();
+      } catch (e) {
+	assertEqual(e.errorMessage, "AQL: k Shortest Paths only has one return variable near 'RETURN a' at position 2:68 (while parsing)");
```

Comment:
@markuspf Maybe it'd be better to only assert the error code, not the exact wording of the message?

Also, I thought a following `LIMIT` should be mandatory?

Reply:
It looks like is there isn't a unique mapping of error codes to messages, and I would quite like to make sure that the correct message is displayed, as before this fix the message referred to "Shortest Path". 

As for the `LIMIT`: At some point @mchacki and Ingo decided that the `LIMIT` wouldn't be mandatory and mention in the documentation that not giving a limit might lead to long waiting times.

The code to check for the presence of `LIMIT` was there, but I subsequently deleted it again, I have no strong feelings about this issue to be honest.